hack/Jenkins/W2L/setup.sh

```diff
@@ -199,7 +199,6 @@ if [ $ec -eq 0 ]; then
 	export TIMEOUT="5m"
 	export DOCKER_HOST="tcp://$ip:$port_inner"
 	export DOCKER_TEST_HOST="tcp://$ip:$port_inner"
-	unset DOCKER_CLIENTONLY
```

Comment:
Can you undo this change - I'll update this script as a separate PR that works in dual mode.